cmd/vclusterctl/cmd/create.go

```diff
@@ -44,12 +45,18 @@ Example:
 vcluster create test --namespace test
 #######################################################
 	`,
-		Args: util.VClusterNameOnlyValidator,
 		RunE: func(cobraCmd *cobra.Command, args []string) error {
+			newArgs, err := util.PromptForMissingArgsIfInteractive(cmd.log, args, "vcluster name")
```

Comment:
Nit: this function name seems excessive. 

Something like `PromptForMissingArgs` or even `PromptForArgs` should be acceptable, and the details about whether something is interactive are more of a doc comment than a name (plus what kind of errors it will return in which circumstances).